graphql-server/go-server/graph/datasource.graphqls

```diff
@@ -103,9 +103,14 @@ type Mutation {
     createDatasource(input: CreateDatasourceInput!): Datasource!
     updateDatasource(input: UpdateDatasourceInput): Datasource!
     deleteDatasource(input: DeleteDatasourceInput): Void
+    createModel(input: CreateModelInput!): Model!
```

Comment:
@zqq454224016 seems we should only have one resolver.  @0xff-dev is this possible to define 
- multiple graphql
- multiple resolvers 
???

Reply:
just `extend type Query`.

example:

- datasource.graphqls
```graphql
type Query {
    query(input: String): String!
}

type Mutation {
    mutation(input: String!): String!
}
```

in you own `some.graphqls`

```graphql
extend type Query {
   someFunc(input: String!): String!
}
```
It will then automatically put the relevant function definitions into a different file.

And I think we can split the current graphqls definitions based on function with another PR.
`entrypoint.graphqls` just defines the overall entry point and some common data structures.
`one.graphqls`, `two.graphqls`... Define interfaces as well as data structures in your own files.